spec/models/cart_spec.rb

```diff
@@ -240,4 +240,64 @@
     it_behaves_like 'with a consecutive reservation', :valid_reservation
     it_behaves_like 'with a consecutive reservation', :checked_out_reservation
   end
+  describe 'check_max_ems' do
+    before(:each) do
+      @em = FactoryGirl.create(:equipment_model, max_per_user: 1)
+      FactoryGirl.create_pair(:equipment_item, equipment_model: @em)
+      @cart.add_item(@em)
+    end
+    it "passes when the reserver doesn't exceed the item limit" do
+      expect(@cart.check_max_ems).to eq([])
+    end
+    it 'passes when the cart contains multiple models' do
+      em_2 = FactoryGirl.create(:equipment_model, max_per_user: 1)
+      FactoryGirl.create_pair(:equipment_item, equipment_model: em_2)
+      @cart.add_item(em_2)
+      expect(@cart.check_max_ems).to eq([])
+    end
+    it 'fails when the reserver exceeds the item limit' do
+      FactoryGirl.create(:valid_reservation,
+                         reserver: User.find_by(id: @cart.reserver_id),
+                         equipment_model: @em)
+      expect(@cart.check_max_ems).not_to eq([])
+    end
+    it 'fails when the reserver exceeds the item limit with an overdue res' do
+      FactoryGirl.create(:overdue_reservation,
```

Comment:
Yeah, `check_max_ems` just counts reservations; it doesn't use the `EquipmentModel` methods because it doesn't actually check availability, just checkout quantity limits